ts/session/conversations/ConversationController.ts

```diff
@@ -262,6 +266,12 @@ export class ConversationController {
     return Array.from(this.conversations.models);
   }
 
+  public getConversationRequests(): Array<ConversationModel> {
+    return Array.from(this.conversations.models).filter(
+      conversation => conversation.isApproved() && !conversation.isBlocked
+    );
+  }
+
```

Comment:
I don't thing this is used.
If used in a react component, just use the selector you have to write